demo/src/index.ts

```diff
@@ -1,7 +1,7 @@
 import { TracingInstrumentation } from '@grafana/agent-tracing-web';
 import { initializeAgent, getWebInstrumentations } from '@grafana/agent-web';
 
-const agent = initializeAgent({
+initializeAgent({
```

Comment:
This was a good idea to allow users to see that the `initializeAgent` returns the agent and they can make use of it afterwards.

Reply:
hmm, ok. 
Need to figure out what to do with it then so it doesn't complain of unused variable.
I didn't want to auto send anything, as it needs to be accounted for all tests then